### ISSUE TITLE
Install as binary not possible?

### ISSUE THREAD

**Issue by MikiGrit** [mike-kfed/lcov2cobertura#10]
Hi,
I wanted to install this crate as binary (with `cargo install lcov2cobertura`). But the command errors with
```bash
error: there is nothing to install in `lcov2cobertura v1.0.3`, because it has no binaries
`cargo install` is only for installing programs, and can't be used with libraries.
To use a library crate, add it as a dependency to a Cargo project with `cargo add`.
```

Is this correct? From readme, it seems it should be possible to use it from command line.

Thanks.

**Comment by mike-kfed** [comment]
yes this is correct, only the library is published. If you want to install the executables, you can clone this repository and run `cargo install --path .` This will install the two binaries under the package name `lcov2xml`. If you know how to package this up as both a library and executables, without introducing useless dependencies to the library users, feel free to submit a PR :)

**Comment by MikiGrit** [comment]
Ok, thanks. For now, I will install it as you suggested.

**Comment by mike-kfed** [comment]
I'd like to keep this open, I am considering making the lcov2xml a separate published crate for ease of install of the executables. Need it as a reminder when I have time again.

**Comment by mike-kfed** [comment]
it is now possible to install the tools using `cargo install lcov2xml`.
